API Blueprint Specification.md

```diff
@@ -277,8 +280,12 @@ What sections can be nested and where depends upon the section in case, as descr
 # II. Sections Reference
 > **NOTE:** Sections marked as "Abstract" serve as a base for other sections and as such they **cannot** be used directly.
 
+<!-- BEGIN Abstract -->
+
+# Abstract
+
 <a name="def-named-section"></a>
-## 1. Named section
+## Named section
```

Comment:
Ah I misread your comment.

I remove all numbers. Again, let me know if you'd like something different.